src/controllers/main/main.ts

```diff
@@ -1357,37 +1356,41 @@ export class MainController extends EventEmitter {
    *   4. for smart accounts, when the Relayer does the broadcast.
    *
    */
-  async broadcastSignedAccountOp(
-    accountOp: AccountOp,
-    estimation: EstimateResult,
-    actionId: AccountOpAction['id']
-  ) {
-    this.broadcastStatus = 'LOADING'
-    this.emitUpdate()
+  async #broadcastSignedAccountOp() {
+    const accountOp = this.signAccountOp?.accountOp
+    const estimation = this.signAccountOp?.estimation
+    const actionId = this.signAccountOp?.fromActionId
```

Comment:
const { accountOp, estimation, fromActionId: actionId } = this.signAccountOp || {}